sonar-db/src/main/resources/org/sonar/db/user/RoleMapper.xml

```diff
@@ -151,24 +135,24 @@
   <select id="countUsersWithPermission" parameterType="map" resultType="int">
     SELECT count(c.id)
     FROM (
-      SELECT DISTINCT ur.user_id as id
-      FROM user_roles ur
-      INNER JOIN users u ON u.id=ur.user_id AND u.active=${_true}
-      WHERE
-        ur.role = #{permission} and ur.resource_id IS NULL
-      UNION ALL
-      SELECT DISTINCT u.id as id
-      FROM users u
-      INNER JOIN groups_users gu ON gu.user_id = u.id
-      INNER JOIN group_roles gr ON gr.group_id = gu.group_id
-      INNER JOIN groups g ON g.id = gu.group_id
-      WHERE
-        gr.role = #{permission}
-        AND gr.resource_id IS NULL
-        AND u.active=${_true}
-        <if test="groupId!=null">
-          AND g.id!=#{groupId}
-        </if>
+    SELECT DISTINCT ur.user_id as id
```

Comment:
argh… this is the configured auto tab of IntelliJ. I'll try to revert this specific change